src/main/java/frc/lib/refvalues/RefValue.java

```diff
@@ -0,0 +1,17 @@
+package frc.lib.refvalues;
+
+public class RefValue<T> {
```

Comment:
Maybe you should put a comment of something that says that ref is a short for referance?

Reply:
I think Ref is popular enough that I don't really need to write it, but anyway I plan on adding Javadoc for everything so I will include this when I get to it.